### PR TITLE
5.7 beta docs

### PR DESCRIPTION
**Pull request by azhebel**

### CODE REVIEW

File: doc/source/install/compile.rst [hunk 1]
```diff
@@ -69,29 +71,40 @@ You will likely have all or most of the packages already installed. If you are n
...
-   .. note:: The directory for Galera repository must be named ``percona-xtradb-cluster-galera``.
+      cd percona-xtradb-cluster
+      git clone https://github.com/percona/galera percona-xtradb-cluster-galera
 
-3. Run the build script :file:`./build-ps/build-binary.sh`. By default, it will build into the current directory, but you can specify another target output directory. For example, if you want to build into :file:`./pxc-build`, run the following:
+   .. note:: The directory for Galera repository
+      must be named ``percona-xtradb-cluster-galera``.
 
-   .. code-block:: bash
+4. Check out the ``rel-3.14.2`` branch.
```
Comment (kbauskar) [inline doc/source/install/compile.rst]: s/rel-3.14.2/3.x/gc

File: doc/source/install/index.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+the default storage engine is InnoDB (MyISAM has only experimental support),
+and set the mode of InnoDB autoincrement locks to ``2``.
+The following is an example of the :file:`my.cnf` file:
+
+.. code-block:: none
+
+   [mysqld]
+
+   wsrep_provider=/usr/lib64/libgalera_smm.so
+   wsrep_cluster_address=gcomm://10.11.12.206
```
Comment (kbauskar) [inline doc/source/install/index.rst:44]: server-id is not more default in MySQL-5.7.
If user want to use log-bin then server-id has to be explicitly specified.

log-bin=mysql-bin

# server id should be unique for each node of pxc-cluster

server-id=1

File: doc/source/install/index.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+===================
+
+It is recommended to install |PXC| from official Percona repositories
+using the corresponding tool for your system:
+
+* :ref:`Install using apt <apt>` if you are running Debian or Ubuntu
+* :ref:`Install using yum <yum>` if you are running Red Hat Enterprise Linux
+  or CentOS
+
+.. note:: You can also `download packages <https://www.percona.com/downloads/Percona-XtraDB-Cluster-56/LATEST/>`_ from the Percona website and install them manually using :command:`dpkg` or :command:`rpm`.
```
Comment (kbauskar) [inline doc/source/install/index.rst]: Why is the link still pointing us to 56? 
We should have 57 link ?

File: doc/source/limitation.rst [hunk 4]
```diff
@@ -1,38 +0,0 @@
-.. _limitations:
```
Comment (kbauskar) [inline doc/source/limitation.rst:1]: We got rid of this but this is much needed and still stand valid for PXC-5.7

File: doc/source/manual/xtrabackup_sst.rst [hunk 5]
```diff
@@ -94,7 +94,7 @@ Used to enable and specify SST encryption mode:
 
    :Values: AES128, AES192, AES256
 
-Used to specify the SST encryption algorithm. It uses the same values as the ``--encryption`` option for XtraBackup (see `here <http://www.percona.com/doc/percona-xtrabackup/2.3/innobackupex/encrypted_backups_innobackupex.html>`_). The ``encrypt-algo`` option is considered only if :option:`encrypt` is set to ``1``.
+Used to specify the SST encryption algorithm. It uses the same values as the ``--encryption`` option for XtraBackup (see `this document <http://www.percona.com/doc/percona-xtrabackup/2.3/innobackupex/encrypted_backups_innobackupex.html>`_). The ``encrypt-algo`` option is considered only if :option:`encrypt` is set to ``1``.
```
Comment (kbauskar) [inline doc/source/manual/xtrabackup_sst.rst]: Just for reference not sure if it is mentioned anywhere.

PXC-5.7.12 can operate with latest version that is 2.4.3 of XB.
PXC-5.7.12 shouldn't be used with older version.

File: doc/source/release-notes/release-notes_index.rst [hunk 6]
```diff
@@ -1,23 +1,10 @@
...
-   Percona-XtraDB-Cluster-5.6.22-25.8
-   Percona-XtraDB-Cluster-5.6.21-25.8
-   Percona-XtraDB-Cluster-5.6.20-25.7
-   Percona-XtraDB-Cluster-5.6.19-25.6
-   Percona-XtraDB-Cluster-5.6.15-25.5
-   Percona-XtraDB-Cluster-5.6.15-25.4
-   Percona-XtraDB-Cluster-5.6.15-25.3
-   Percona-XtraDB-Cluster-5.6.15-25.2
-   Percona-XtraDB-Cluster-5.6.14-25.1
+   Percona-XtraDB-Cluster-5.7.11-4beta
```
Comment (kbauskar) [inline doc/source/release-notes/release-notes_index.rst]: I assume the naming convention here is "xxxx-5.7_5.7.11-25.14.2-beta" .. please re-confirm with RE team and see what needs to be correctly mentioned.

File: doc/source/release-notes/release-notes_index.rst [hunk 7]
```diff
@@ -1,23 +1,10 @@
...
+   Percona-XtraDB-Cluster-5.7.12-25.16
```
Comment (kbauskar) [inline doc/source/release-notes/release-notes_index.rst]: Same as above. This is beta release too.

**Comment by hrvojem** [comment]
LGTM

**Pull request merged**